koans/triangle.rb

```diff
@@ -15,8 +15,15 @@
 #
 def triangle(a, b, c)
   # WRITE THIS CODE
+  raise TriangleError.new, "Triangle Error" if [a, b, c].min <= 0
```

Comment:
without `.new`. And add an empty line after `raise` and `return`
```suggestion
  raise TriangleError, "Triangle Error" if [a, b, c].min <= 0

```